src/utils/decancer.rs

```diff
@@ -1,14 +1,29 @@
 extern crate decancer;
 
+use censor::*;
 use tracing::{error, info, trace};
 
-use super::*;
+use super::{*, bot::get_data_lock};
 
 pub async fn cure(
     ctx: &serenity::Context,
     old_if_available: &Option<serenity::Member>,
     member: &serenity::Member,
 ) {
+    let data_lock = get_data_lock(ctx).await;
+    let additions = &data_lock.read().await.configuration.general.censor.additions;
+    let removals = &data_lock.read().await.configuration.general.censor.removals;
+
+    let mut censor = Standard;
```

Comment:
The code above can be moved below the check for bots. Additionally the for loops are ran every time the method is called, you could lazily initialize the two arrays statically, so that it has to only ever run once, but then you'd need to handle the case when calling /reload.